src/watts/plugin_accert.py

```diff
@@ -0,0 +1,98 @@
+# SPDX-FileCopyrightText: 2022 UChicago Argonne, LLC
+# SPDX-License-Identifier: MIT
+
+from datetime import datetime
+from pathlib import Path
+import os
+import sys
+from typing import List, Optional
+import pandas as pd
+
+from .fileutils import PathLike
+from .parameters import Parameters
+from .plugin import PluginGeneric
+from .results import Results
+
+
+class PluginACCERT(PluginGeneric):
+    """Plugin for running ACCERT
+
+    Parameters
+    ----------
+    template_file
+        Templated ACCERT input
+    extra_inputs
+        List of extra (non-templated) input files that are needed
+    extra_template_inputs
+        Extra templated input files
+    show_stdout
+        Whether to display output from stdout when ACCERT is run
+    show_stderr
+        Whether to display output from stderr when ACCERT is run
+
+    Attributes
+    ----------
+    executable
+        Path to ACCERT executable
+
+    """
+
+    def __init__(self, template_file: str,
+                 extra_inputs: Optional[List[str]] = None,
+                 extra_template_inputs: Optional[List[PathLike]] = None,
+                 show_stdout: bool = False, show_stderr: bool = False):
+        try:
+            executable = Path(os.environ['ACCERT_DIR']) / 'Main.py'
+        except KeyError:
+            raise OSError("ACCERT_DIR environment variable needs to be set which contains the Main.py")
```

Comment:
You should provide an `executable` argument that gives the user a way of specifying the path to `Main.py` without necessarily having to set an environment variable (you can look at the PyARC plugin as an example of this).